Apollo/Components/CopyOffset.cs

```diff
@@ -82,5 +89,35 @@ public class CopyOffset: UserControl {
             _viewer.X = x;
             _viewer.Y = y;
         }
+    
+        public void SetAngle(double angle, bool isRadians = true){
+            if(isRadians){
+                AngleDial.RawValue = angle/Math.PI*180;
+            } else {
+                AngleDial.RawValue = angle;
+            }
+        }
+        
+        public void Angle_Changed(Dial sender, double angle, double? old){
+            int index = _copy.Offsets.IndexOf(_offset);
+            
+            if(old != null && old.Value != angle){
+                List<int> path = Track.GetPath(_copy);
+                
+                double a = angle;
+                double o = old.Value;
+
+                Program.Project.Undo.Add($"Copy Angle {index + 1} Changed to {angle}°", () => {
+                    Copy copy = ((Copy)Track.TraversePath(path));
+                    copy.Offsets[index].Angle = old.Value/180*Math.PI;
```

Comment:
you're using `old` and `angle` instead of `a` and `o`. usually they're called `u` and `r` instead of `a` and `o`, anyway